demo/atom/backToTop/playground

```diff
@@ -0,0 +1,247 @@
+const iconTop = () => (
+  <svg width="100%" height="100%" viewBox="0 0 274 352">
+    <g id="Page-1" stroke="none" fill="none">
+      <g id="arrow_c_up_icon_128" fill="#000000">
+        <path
+          d="M9.4,109.3 L114.4,9 C120.2,3 128.1,0 136.8,0 C145.5,0 153.3,3 159.2,9 L264.6,109.3 C277.1,121.2 277.1,140.6 264.6,152.5 C252.1,164.4 231.9,164.4 219.4,152.5 L169,104.4 L169,321.4 C169,338.3 154.7,352 137,352 C119.3,352 105,338.3 105,321.4 L105,104.4 L54.6,152.6 C42.1,164.5 21.9,164.5 9.4,152.6 C-3.1,140.6 -3.1,121.3 9.4,109.3 Z"
+          id="Shape"
+        />
+      </g>
+    </g>
+  </svg>
+)
+
+return (
+  <div>
```

Comment:
the example is to short for big screens, you should make it longer

Reply:
Solved w/ a button in the Demo that adds paragraphs on demand